docs/integration/alerts-googlechat-integration.md

```diff
@@ -0,0 +1,50 @@
+title: Google Chat Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Google Chat. Use them together to get the information about each notification quickly and on any device.
+
+## In Google Chat
+
+**1.** Create a new space that will be used to receive notifications from Sematext, or use an existing one.
+
+On the spaces tab, click the **+** icon:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - Add Space" src="../../images/integrations/create-googlechat-integration-add.png" title="Create Google Chat Integration - Add Space">
+
+**2.** Select **Create Space** option:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - New Space" src="../../images/integrations/create-googlechat-integration-new-space.png" title="Create Google Chat Integration - New Space">
+
+**3.** Give your space a name and click **Create**:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - Create Space" src="../../images/integrations/create-googlechat-integration-create.png" title="Create Google Chat Integration - Create Space">
+
+**4.** Open the drop down menu with your space's name and select **Manage webhooks**:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - Manage Webhooks" src="../../images/integrations/create-googlechat-integration-manage-webhooks.png" title="Create Google Chat Integration - Manage Webhooks">
+
+**5.** Give your webhook a name (and optionally an avatar) and click **Save**:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - Save Webhook" src="../../images/integrations/create-googlechat-integration-save-webhooks.png" title="Create Google Chat Integration - Save Webhook">
+
+**6.** Copy the webhook URL using the copy button:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - Copy Webhook" src="../../images/integrations/create-googlechat-integration-copy-webhooks.png" title="Create Google Chat Integration - Copy Webhook">
+
+With the **Incoming Webhook** configured we can now use the **Webhook URL** to configure the Google Chat notification hook in Sematext.
+
+## In Sematext
+
+**1.** Navigate to [Notification Hooks](https://apps.sematext.com/ui/hooks/create) (in [EU](https://apps.eu.sematext.com/ui/hooks/create)) and select Google Chat card to create a new Google Chat notification hook.
+
+<img class="content-modal-image" alt="Create Google Chat Integration - New Webhook" src="../../images/integrations/create-googlechat-integration-new-hook.png" title="Create Google Chat Integration - New Webhook">
+
+**2.** Give your webhook a name and add your Google Chat **Webhook URL**.
```

Comment:
webhook ==> Notification Hook
(and the same sort of changes further down)

Reply:
Here on line 40, since it comes from Google and they call it Webhook, I'd keep it. 
For everything on our side, I'm making the change. What do you think?